app/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/slf4j/Slf4jLoggerTest.groovy

```diff
@@ -0,0 +1,68 @@
+package gov.hhs.cdc.trustedintermediary.external.slf4j
+
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+
+import org.slf4j.event.Level
+import org.slf4j.MarkerFactory
+
+import spock.lang.Specification
+
+
+class Slf4jLoggerTest extends Specification {
```

Comment:
I could not figure out how to set the log level for slf4j at runtime to test DEBUG and TRACE. Maybe if there is a way to create a `simplelogger.properties` files for the test folder like the one in `app/src/main/resources/simplelogger.properties`?

Reply:
Nevermind, I found out how to add the properties file for the tests and added the missing tests